### PR TITLE
dot release notes

### PR DESCRIPTION
**Pull request by sohailta**

### CODE REVIEW

File: Teams/room-systems/srs2-release-note.md [hunk 1]
```diff
@@ -53,6 +54,14 @@ This article discusses cumulative improvements in Microsoft Teams Rooms.
 
 ## Microsoft Teams Rooms feature introduction and issue resolution
 
+### 4.3.33.0 (1/10/2020)
+
+Introduced in this update:
+
+- A fix for a Window resizing/flickering issue that's seen in certain configurations
+- Calendar processing for 3rd-party meetings removed
```
Comment (JohanFreelancer9) [inline Teams/room-systems/srs2-release-note.md]:
```suggestion
- Calendar processing for third-party meetings removed
```

Comment (dstrome) [reply Teams/room-systems/srs2-release-note.md]: @sohailta - can you please accept and merge this change? I don't have permissions to do so in your repo.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/room-systems/srs2-release-note.md (hunk 1 above)
Comment (LanaChin) [reply Teams/room-systems/srs2-release-note.md]: I updated the MD with the change that @JohanFreelancer9 suggested, and committed it.

**Pull request merged**